src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -666,6 +668,36 @@ public static void setDistanceModelUpdateUrl(String url) {
         distanceModelUpdateUrl = url;
     }
 
+    /**
+     * Default class for rssi filter/calculation implementation
+     */
+    protected static Class rssiFilterImplClass = RunningAverageRssiFilter.class;
```

Comment:
instead of Class, can we make this RssiFilter?  (Same goes for the getter and setter)